packages/front-end/pages/metric/[mid].tsx

```diff
@@ -133,11 +136,18 @@ const MetricPage: FC = () => {
   }
 
   const metric = data.metric;
-  const canDuplicateMetric = permissionsUtil.canCreateMetric(metric);
+  const canDuplicateMetric =
+    metric.managedBy === "admin"
+      ? permissionsUtil.canCreateOfficialResources(metric)
+      : permissionsUtil.canCreateMetric(metric) && !metric.managedBy;
   const canEditMetric =
-    permissionsUtil.canUpdateMetric(metric, {}) && !metric.managedBy;
+    metric.managedBy === "admin"
+      ? permissionsUtil.canUpdateOfficialResources(metric, {})
+      : permissionsUtil.canUpdateMetric(metric, {}) && !metric.managedBy;
   const canDeleteMetric =
-    permissionsUtil.canDeleteMetric(metric) && !metric.managedBy;
+    metric.managedBy === "admin"
+      ? permissionsUtil.canDeleteOfficialResources(metric)
+      : permissionsUtil.canDeleteMetric(metric) && !metric.managedBy;
```

Comment:
Same here.  Fix these permission checks to always check for `canCreateMetric`, etc. in addition to `canCreateOfficialResources`.  

```ts
const canDeleteMetric =
    permissionsUtil.canDeleteMetric(metric) && (
      // Either not Official
      !metric.managedBy
      || (
        // Or it's Official and the user has permission to delete anyway
        metric.managedBy === "admin" && 
        permissionsUtil.canDeleteOfficialResources(metric)
      )
    )
```